### PR TITLE
Integration tests for the user account/my_preferences module

### PR DESCRIPTION
**Pull request by nicolasbrossard**
Integration tests for the user account/my_preferences module

### CODE REVIEW

File: modules/user_accounts/test/user_accountsTest.php [hunk 1]
```diff
@@ -36,34 +62,313 @@ function testUserAccountsDoespageLoad()
...
     {
         $this->safeGet($this->url . "/user_accounts/my_preferences/");
-        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $bodyText = $this->webDriver->findElement(
+            WebDriverBy::cssSelector("body")
+        )->getText();
         $this->assertContains("My Preferences", $bodyText);
     }
+    /**
+     * Tests that searching for users using thei user IDs works
```
Comment (Jkat) [inline modules/user_accounts/test/user_accountsTest.php]: their*

File: modules/user_accounts/test/user_accountsTest.php [hunk 2]
```diff
@@ -6,14 +6,39 @@
...
+ * @category Test
+ * @package  Loris
+ * @author   Nicolas Brossard <nicolasbrossard.mni@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class UserAccountsIntegrationTest extends LorisIntegrationTest
 {
+    private static $_UNIT_TESTER = array(
+                                    'Data Coordinating Center',
```
Comment (Jkat) [inline modules/user_accounts/test/user_accountsTest.php]: Should this be 'DCC'?

File: modules/user_accounts/test/user_accountsTest.php [hunk 3]
```diff
@@ -36,34 +62,313 @@ function testUserAccountsDoespageLoad()
...
+    }
+    /**
+     * Tests various user account edit operations.
+     *
+     * @return void
+     */
+    function testUserAccountEdits()
+    {
+        $this->_verifyUserModification(
+            'user_accounts',
```
Comment (Jkat) [inline modules/user_accounts/test/user_accountsTest.php]: Should these urls be instead something like 'user_accounts/edit_user'?

Comment (nicolasbrossard) [inline modules/user_accounts/test/user_accountsTest.php]: Actually no. I access the user accounts page to check that the modifications done previously we successfully done.

**Pull request merged**